src/games/gameitem.py

```diff
@@ -209,11 +261,27 @@ def _game_observers(self, game):
         return [game.to_player(name) for name in game.observers
                 if game.is_connected(name)]
 
+    def _game_sim_mods(self, game):
+        if not game.sim_mods:
```

Comment:
Another example of oddly defensive programming.

What's the contract of the `game` object? It would appear sensible to make it always have the `sim_mods` field, and simply set it to empty-list when there are no mods. Having to check for existence of fields all the time is a really good bug factory.